<launch>
  <param name="use_sim_time" value="true" /> #be sure to run --clock

  <!-- Config params -->
  <arg name="config_filename" default="$(find loam)/config/euroc_maplabIntegrator.yaml" />

  <!-- LOAM Nodelet Manager -->
  <node pkg="nodelet" type="nodelet" name="loam_nodelet_manager" args="manager" output="screen" clear_params="true"/> #launch-prefix="gdb -ex run --args" />

  <!-- Laser Graph Integrator Nodelet -->
  <node pkg="nodelet" type="nodelet" name="laser_graph_integrator_nodelet" args="load laser_graph_integrator_nodelet loam_nodelet_manager" output="screen">
    <rosparam file="$(arg config_filename)" command="load"/>
    <remap from="odometry" to="/euroc_3/rovioli/odom_T_M_I" />
    <remap from="absolute_constraints" to="/anymal/absolute_reference" />
    <remap from="anchor_constraints" to="/graph_client/anchor_nodes" />
    <remap from="submap_constraints" to="/graph_client/relative_nodes" />
  </node>
</launch>
